test/integration.test.js

```diff
@@ -22,11 +23,20 @@ describe('css-vendor', () => {
       expect(supportedProperty('display')).to.be('display')
     })
 
-    it('should prefix if needed', () => {
-      const prop = supportedProperty(`${prefix.css}animation`)
-      if (prop !== 'animation') {
-        expect(prop).to.be(`${prefix.css}animation`)
-      }
+    browser("requires prefix for transform property", {
```

Comment:
I see what you mean. I'll checkout how complicated it is to use the api directly.